demisto_sdk/commands/update_xsoar_config_file/tests/update_xsoar_config_file_test.py

```diff
@@ -35,6 +35,10 @@ def temp_dir():
         rmtree(temp)
 
 
+def json_file(config_file):
```

Comment:
what's the just of this function?

Reply:
I don't know probably added by mistake....